### DISCUSSION TITLE
How to link registered customer and their previous orders ?

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [medusajs/medusa#2826] [Q&A] [answered]
Hey, I just wanted to know how can merge customer into one ?

<img width="499" alt="Capture d’écran 2022-12-19 à 02 02 46" src="https://user-images.githubusercontent.com/32040951/208329370-e057e377-e0bf-40f8-9538-4df53f2e6a07.png">

**Comment by pKorsholm** [comment, marked as answer]
Hi @MatteoGauthier 

We dont have a 'merge' operation on customers at the moment so a bit of manual work is required. We have however just introduced the notion of transferring orders between customers. 

We've just merged the update to the medusa admin so you need to re-pull in order to merge customers using the admin UI and install the latest medusa (1.7.0). 

In order to merge two customers you would manually update the customer information between them, and then manually transfer orders using the transfer order modal from the customer overview (tranferring ordres to the receiving customer):

![image](https://user-images.githubusercontent.com/88927411/208402168-1fd72e81-7f0c-4a38-a9c9-b339c3df8889.png)
